src/main/java/core/basesyntax/service/ReaderImpl.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.service;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+
+public class ReaderImpl implements Reader {
+    @Override
+    public List<String> readFromFile(String filePath) {
+        List<String> content;
+        try {
+            content = Files.readAllLines(Paths.get(filePath));
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read from file");
+        }
+        content.remove(0);
```

Comment:
what if we want to reuse our reader? 
let's skip the 1st line in a different place